app/src/main/java/com/compscieddy/meetinthemiddle/GroupActivity.java

```diff
@@ -498,6 +500,39 @@ public void onMapClick(LatLng point) {
   }
 
   @Override
+  public void onMapDrag() {
+    if (!collapsed){
+      Display display = getWindowManager().getDefaultDisplay();
+      Point size = new Point();
+      display.getSize(size);
+      int height = size.y;
+
+      ResizeAnimation resizeAnimation;
+
```

Comment:
Don't copy & paste code! Please refactor out this code to make it okay with the code below